misc/efi-stub/Makefile

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright (c) 2011 - 2021, Intel Corporation
```

Comment:
Shouldn't all these files use the approved ACRN license/copyright header? @NanlinXie